bin/reset-e2e-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
+
+# Include useful functions.
+. "$(dirname "$0")/includes.sh"
+
+# Set up WordPress site used for end-to-end (e2e) tests.
+. "$(dirname "$0")/install-wordpress.sh" --e2e_tests
```

Comment:
If we can slim this down to just deleting posts and such I'm game, but @aduth mentioned in #8010 we might want to uninstall plugins and such. At that point it seemed easier to just re-install at the outset of every test run.

Reply:
A reinstall is pretty quick, and it guarantees you're getting a fresh state.